src/compute/src/render/top_k.rs

```diff
@@ -847,7 +847,7 @@ pub mod monoids {
     use timely::container::columnation::{Columnation, Region};
 
     /// A monoid containing a row and an ordering.
-    #[derive(Eq, PartialEq, Debug, Clone, Serialize, Deserialize, Hash)]
+    #[derive(Eq, PartialEq, Debug, Clone, Serialize, Deserialize, Hash, Default)]
```

Comment:
What necessitated this change?

Reply:
Something deep down in the bowels of Differential gained a `Default` constraint.